packages/core/test/batch/batch-request-serializer.spec.ts

```diff
@@ -35,6 +36,24 @@ describe('batch request serializer', () => {
       ).toMatchSnapshot();
     });
 
+    it('serializes getAll request with absolute sub request path', () => {
+      expect(
+        serializeRequest(TestEntity.requestBuilder().getAll(), {
+          subrequestPath: 'absolute',
+          destination: { url: 'http://example.com' }
+        })
+      ).toMatchSnapshot();
+    });
+
+    it('serializes getAll request with entity relative sub request path', () => {
+      expect(
+        serializeRequest(TestEntity.requestBuilder().getAll(), {
+          subrequestPath: 'entityRelative',
+          destination: { url: 'http://example.com' }
+        })
+      ).toMatchSnapshot();
```

Comment:
Minor: I prefer in such cases a `matches` with a regex of the kind `.*GET /A_TestEntity?$format=json HTTP/1.1.*`  and `.*GET http://example.com/sap/opu/odata/sap/API_TEST_SRV/A_TestEntity?$format=json HTTP/1.1.*`. With this it is easier to see what is the actual assertion, namely the full path or not full paht. In the snapshot this is hard to tell.